acquisitions/src/main/resources/thunderjet/mod-gobi/features/gobi-api-tests.feature

```diff
@@ -80,3 +80,214 @@ Feature: mod-gobi api tests
     And headers { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': '*/*' }
     When method DELETE
     Then status 200
+
+  Scenario: Try to send invalid requests and check responses
+    * def valid_mapping = read('classpath:samples/mod-gobi/unlisted-print-serial-valid.json')
+    * def invalid_mapping_type = read('classpath:samples/mod-gobi/unlisted-print-serial-custom.json')
+    * def invalid_mapping_field = read('classpath:samples/mod-gobi/unlisted-print-serial-2.json')
+    * def invalid_mapping_translation = read('classpath:samples/mod-gobi/unlisted-print-serial-1.json')
```

Comment:
lets create new folder and move invalid mappings in that folder, it would be more clear to understand what mappings are not valid